src/azul/chalice.py

```diff
@@ -189,15 +193,35 @@ def _api_gateway_context_middleware(self, event, get_response):
         finally:
             config.lambda_is_handling_api_gateway_request = False
 
+    hsts_max_age = 60 * 60 * 24 * 365 * 2
+
+    # These headers are added to every response from an AWS Lambda function.
+    #
+    security_headers = {
+        'Strict-Transport-Security': jw(f'max-age={hsts_max_age};',
+                                        'includeSubDomains;',
+                                        'preload'),
+        'X-Content-Type-Options': 'nosniff',
+        'X-Frame-Options': 'DENY',
+    }
+
+    # These headers are added to 4XX and 5XX responses from the API Gateway, for
+    # example if an invalid URL was requested (404), or if a request took too
+    # long to complete (504).
```

Comment:
```suggestion
    # These headers are added to 4XX and 5XX responses from API Gateway, for
    # example if an invalid URL was requested (404), or if a request took too
    # long to complete (504).
```

It's still not clear from these comments as to why these definitions are separate. You explain the "what" and the "how", but not they "why": Why can't `Content-Security-Policy`, `Referrer-Policy` and `X-XSS-Protection` be part of all Lambda-generated responses?

Reply:
I see no reason why `Referrer-Policy` and `X-XSS-Protection` can't be included in all responses. `Content-Security-Policy` however (in it's current form) is only suitable for an error response, so will remain separate. (https://github.com/DataBiosphere/azul-private/issues/6 will follow this PR to add the CSP header to all responses.)
In the latest fixup commit I've moved the two headers mentioned above, and rewritten the comments.